templates/CWP/CWP/PageTypes/Layout/NewsHolder.ss

```diff
@@ -13,6 +15,8 @@
             <% include NewsFilterContext %>
         </div>
 
+        <% include NewsFilterContext %>
```

Comment:
This look like a duplicate of line 13. Is this intended?

Reply:
Removed!